docs/development/index.rst

```diff
@@ -210,9 +218,11 @@ the error is being raised for.
 
 Unit testing
 ------------
-Blocks uses unit testing to ensure that individual parts of the library behave
-as intended. It's also essential in ensuring that parts of the library are not
-broken by proposed changes.
+Blocks and Fuel use unit testing to ensure that individual parts of
+the library behave as intended. It's also essential in ensuring that
+parts of the library are not broken by proposed changes. Since Blocks
+depends on Fuel, it is important to make sure changes in Fuel do not
```

Comment:
It doesn't really depend on Fuel. I would say they designed to be used together.